File: code-coach/free/easy/cheer_creator.cpp
#include <iostream>
using namespace std;

int main() {
    int yard;
    cin >> yard;

    if (yard > 10) {
        cout << "High Five\n";
    } else if (yard < 1) {
        cout << "shh\n";
    } else {
        for(int i = 0; i < yard; i++) {
            cout << "Ra!";
        }
    }

    return 0;
}

File: code-coach/free/easy/cheer_creator.cs
using System;

namespace Sololearn {
    class Program {
        static void Main(string[] args) {
            int yard;
            if (int.TryParse(Console.ReadLine(), out yard)) {
                if (yard > 10) {
                    Console.WriteLine("High Five");
                }
                else if (yard < 1) {
                    Console.WriteLine("shh");
                }
                else {
                    for (int i = 0; i < yard; i++) {
                        Console.Write("Ra!");
                    }
                }
            }
        }
    }
}